ister.py

```diff
@@ -453,10 +453,18 @@ def copy_os_swupd(args, template, target_dir):
     cmd = "swupd verify --install"
     cmd += " --path={0}".format(target_dir)
     cmd += " --manifest={0}".format(template["Version"])
-    if args.contenturl:
-        cmd += " --contenturl={0}".format(args.contenturl)
-    if args.versionurl:
-        cmd += " --versionurl={0}".format(args.versionurl)
+    if args.contenturl or template.get("MirrorURL"):
```

Comment:
So this is for the swupd command, it should only look at the arg.contenturl

Reply:
Well, one of the main points of this feature (when using the GUI) is to allow the user to modify where to get content during the build (for testing, or to improve build speed on a local mirror perhaps) - if I don't honor this on swupd cmd line - it will likely be confusing to the user.

Reply:
My thought is to have the gui update the args passed to ister as well as the template rather than have ister use the args or the template.